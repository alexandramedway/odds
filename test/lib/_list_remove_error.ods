do list_remove(4, [0, 1, 2, 3])
