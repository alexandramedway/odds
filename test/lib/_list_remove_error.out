list_remove: List index out of range
